dom-manipulation/main.js

```diff
@@ -0,0 +1,25 @@
+var $timesClicked = 0;
+var $hotButton = document.querySelector('.hot-button');
+var $clickCount = document.querySelector('.click-count');
+
+$hotButton.addEventListener('click',
+  function (event) {
+    $timesClicked += 1;
+    // console.log($timesClicked);
+    $clickCount.textContent = 'Clicks:' + $timesClicked;
+    // document.getElementsByClassName('.cold');
+    if ($timesClicked <= 4) {
+      $hotButton.className = 'hot-button cold';
+    } else if ($timesClicked <= 7) {
+      $hotButton.className = 'hot-button cool';
+    } else if ($timesClicked <= 10) {
+      $hotButton.className = 'hot-button tepid';
+    } else if ($timesClicked <= 13) {
+      $hotButton.className = 'hot-button warm';
+    } else if ($timesClicked <= 16) {
```

Comment:
The conditionals should actually just be less than, rather than less than or equal to.

Reply:
Right.  I do see that now in the assignment instructions.  Thx.